src/base/bittorrent/torrenthandleimpl.h

```diff
@@ -53,35 +54,26 @@ namespace BitTorrent
     class Session;
     struct AddTorrentParams;
 
-    struct CreateTorrentParams
+    struct LoadTorrentParams
     {
-        CreateTorrentParams() = default;
-        explicit CreateTorrentParams(const AddTorrentParams &params);
+        lt::add_torrent_params ltAddTorrentParams {};
```

Comment:
Seems `{}` is redundant or you should add it at `struct LoadedMetadataHandle` too, to keep them consistent.

Reply:
>you should add it at `struct LoadedMetadataHandle` too, to keep them consistent.

Added there as well.
Although MSVC/GCC compiles successfully without `{}`, the analyzer used in QtCreator throws an error (something related to the fact that the default constructor of `lt::add_torrent_params` is declared explicit). Since the analyzer is based on Clang, I assume that Clang will fail to compile this code so I prefer to keep it without this error.

Other comments are addressed too.